src/errors.js

```diff
@@ -1,8 +1,11 @@
+/* eslint-disable prefer-rest-params*/
+/* eslint-disable no-var */
+
 var NO_DATA_MSG = 'No data'
```

Comment:
Why this is left as `var` and `/* eslint-disable no-var */` is added?

Reply:
Because I can't export const's for some reason, I tried one way or another, but it couldn't get exported, with vars it works just fine (The vars are used in other files)

Reply:
Can't you do `export const ...`?

Reply:
No, it doesn't work

Reply:
Tried it with module.exports = {...} and with export... but doesn't work in any of these ways

Reply:
The const is just undefined in those cases

Reply:
https://stackoverflow.com/a/40596877
maybe this?

Reply:
I saw this actually, but I don't know if it will work, mainly because the file get exported, but somehow ends up  undefined.